### ISSUE TITLE
VSCode Extension - Custom connectors add connector location to Api Id 

### ISSUE THREAD

**Issue by zzznz27** [Azure/LogicAppsUX#5550]
### Describe the Bug with repro steps 

1. Create a new logic app workflow in VSCode 
2. Add an action from a custom connector 
3. Set up a new connection 
4. Observe in the connections.json, the new connection created is in the format: 

```json
{
  "Custom-Connector_Connection": {
        "api": {
          "id": "/subscriptions/@{appsetting('WORKFLOWS_SUBSCRIPTION_ID')}/resourceGroups/@{appsetting('WORKFLOWS_RESOURCE_GROUP_NAME')}/providers/@{appsetting('WORKFLOWS_LOCATION_NAME')}/customApis/custom-connector-name"
        },
       ...
  }
}
```

When compared to the ID of the Custom Connector there is no reference to the location

```
@{appsetting('WORKFLOWS_LOCATION_NAME')}
```

 instead it is:

```
/subscriptions/@{appsetting('WORKFLOWS_SUBSCRIPTION_ID')}/resourceGroups/@{appsetting('WORKFLOWS_RESOURCE_GROUP_NAME')}/providers/customApis/custom-connector-name
```

### What type of Logic App Is this happening in?

Standard (VSCode)

### Which operating system are you using?

Windows

### Are you using new designer or old designer

New Designer

### Did you refer to the TSG before filing this issue? [https://aka.ms/lauxtsg](<https://aka.ms/lauxtsg>)

Yes

### Workflow JSON

*No response*

### Screenshots or Videos

*No response*

### Browser

VSCode

### Additional context

*No response*

**Comment by makesleek** [comment]
For API Connections the uri should be something like this:

`/subscriptions/@{appsetting('WORKFLOWS_SUBSCRIPTION_ID')}/resourceGroups/@{appsetting('WORKFLOWS_RESOURCE_GROUP_NAME')}/providers/Microsoft.Web/customApis/<myapi>`

The location reference shouldn't be there at all.

As a workaround you can manually edit the value in `connections.json` and replace the location part with the actual provider.

**Comment by ccastrotrejo** [comment]
Hi @zzznz27 sorry for the late reply on this. I just noticed it. This usually happens as we parameterize all the connections. 

- Does your `local.settings.json` file include the property `WORKFLOWS_LOCATION_NAME`?
- What is the correct form of the connection id?

Here is a small blog that talks about it. [Parameterizing Managed Connections with Logic Apps Standard](https://techcommunity.microsoft.com/blog/integrationsonazureblog/parameterizing-managed-connections-with-logic-apps-standard/3660679)

**Comment by zzznz27** [comment]
This issue is related to the incorrect formatting rather than the parameterisation 

The location reference should not exist in the custom connection of but it get auto populated 

You can just remove it as a work around but it would be good to get the id of the custom connector right from the initial setup

**Comment by ccastrotrejo** [comment]
@zzznz27 Yep, that makes sense. It seems that for custom connectors connections id shouldn't have that extra parameterization. I will take a look on it. Thanks!
